src/cmd/INIT/package.mk

```diff
@@ -30,7 +30,7 @@
  *	:LICENSE: license-class-pattern
  *	:CATEGORY: category-id ...
  *	:COVERS: package ...
- *	:REQURES: package ...
```

Comment:
Ditto here.

Reply:
Ditto reply (see above).

Reply:
Ditto, that it's fine as it's only in a comment.